go-controller/pkg/node/node.go

```diff
@@ -223,54 +223,39 @@ func setupOVNNode(node *kapi.Node) error {
 	return nil
 }
 
-func isOVNControllerReady(name string) (bool, error) {
+func (n *OvnNode) isOVNControllerReady() (bool, error) {
+	// check nodes connection status
 	runDir := util.GetOvnRunDir()
-
 	pid, err := ioutil.ReadFile(runDir + "ovn-controller.pid")
 	if err != nil {
 		return false, fmt.Errorf("unknown pid for ovn-controller process: %v", err)
 	}
-
-	err = wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
-		ctlFile := runDir + fmt.Sprintf("ovn-controller.%s.ctl", strings.TrimSuffix(string(pid), "\n"))
-		ret, _, err := util.RunOVSAppctl("-t", ctlFile, "connection-status")
-		if err == nil {
-			klog.Infof("Node %s connection status = %s", name, ret)
-			return ret == "connected", nil
-		}
-		return false, err
-	})
+	ctlFile := runDir + fmt.Sprintf("ovn-controller.%s.ctl", strings.TrimSuffix(string(pid), "\n"))
+	ret, _, err := util.RunOVSAppctl("-t", ctlFile, "connection-status")
 	if err != nil {
-		return false, fmt.Errorf("timed out waiting sbdb for node %s: %v", name, err)
+		return false, fmt.Errorf("could not get connection status for node %s: %w", n.name, err)
```

Comment:
(Having the node name in these errors doesn't seem like it really matters that much? You know it's talking about the local node...)

Reply:
Good point. I updated it.